nixos/modules/programs/gpu-screen-recorder.nix

```diff
@@ -31,7 +31,6 @@ in {
     security.wrappers."gpu-screen-recorder" = {
```

Comment:
I couldn't mark it in github before but I think you need to remove the whole security.wrappers."gpu-screen-recorder" = { section, not just this line.